library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -1099,8 +1099,8 @@ private long getPausedReason(int status) {
         }
 
         private long getErrorCode(int status) {
-            if ((400 <= status && status < DownloadStatus.MIN_ARTIFICIAL_ERROR_STATUS)
-                    || (500 <= status && status < 600)) {
+            if (400 <= status && status < DownloadStatus.MIN_ARTIFICIAL_ERROR_STATUS
```

Comment:
lol no idea either but the PRB wouldn't stop closing the PR until I fixed 3 PMD issues so I removed unnecessary parenthesis